SlicerRadiomicsCLI/SlicerRadiomicsCLI

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python-real
+#!/usr/bin/env python
```

Comment:
I don't think you want this - python-real will be the slicer-specific one that is installed and will be compatible with the packages in your python path when running in the slicer environment. 

One issue for development is that python-real is created in the install step so it's not convenient to test this in a developer build.

Reply:
If that's the case then yes, we should keep `python-real`, but is it then possible to also allow for using the `#!/usr/bin/env python` when working with a built slicer (and not an installed one)?

Reply:
It might be easiest to make this a shell script that invokes the right python executable depending on the environment.

We want to make sure we use [the python that is installed with all the right settings](https://github.com/Slicer/Slicer/blob/03ef3b63ab8f2c17843b4234cb5d944dedc1e99b/CMake/SlicerBlockInstallPython.cmake).

I can do some tests on mac/linux build vs install trees if needed.

Reply:
@pieper, I'm currently working on rebuilding the regular module to use the CLI interface, but this is a good point to follow up. I have now also a Linux VM running (Ubuntu) with both a Slicer build and slicer install running. On windows, building Slicer is still giving me headaches from time to time, but there I toy around a lot with a binary installed tree.

Reply:
Okay sounds good - let me know if you want help testing anything.